tracer.spec

```diff
@@ -1,79 +1,67 @@
 
 %package -n python2-%{name}
 Summary:        %{summary}
 Provides:       %{name} = %{version}-%{release}
 Obsoletes:      %{name} <= %{version}-%{release}
+BuildRequires:  python2-devel
 BuildRequires:  python-sphinx
 BuildRequires:  python-beautifulsoup4
 BuildRequires:  python-psutil
 BuildRequires:  python-pygments
 BuildRequires:  python-lxml
 Requires:       rpm-python
-Requires:       python
 Requires:       python-beautifulsoup4
 Requires:       python-psutil
 %{?python_provide:%python_provide python2-%{name}}
 
 %package -n python3-%{name}
 Summary:        %{summary}
+BuildRequires:  python3-devel
 BuildRequires:  python3-sphinx
 BuildRequires:  python3-beautifulsoup4
 BuildRequires:  python3-psutil
 BuildRequires:  python3-pygments
 BuildRequires:  python3-lxml
 Requires:       rpm-python3
-Requires:       python3
 Requires:       python3-beautifulsoup4
 Requires:       python3-psutil
 %{?python_provide:%python_provide python3-%{name}}
 
 
-%description
-Tracer determines which applications use outdated files and prints them. For
-special kind of applications such as services or daemons, it suggests a standard
-command to restart it. Detecting whether file is outdated or not is based on a
-simple idea. If application has loaded in memory any version of a file
-which is provided by any package updated since system was booted up, tracer
+%global _description \
+Tracer determines which applications use outdated files and prints them. For\
+special kind of applications such as services or daemons, it suggests a standard\
+command to restart it. Detecting whether file is outdated or not is based on a\
+simple idea. If application has loaded in memory any version of a file\
+which is provided by any package updated since system was booted up, tracer\
 consider this application as outdated.
 
-%description -n python2-%{name}
-Tracer determines which applications use outdated files and prints them. For
-special kind of applications such as services or daemons, it suggests a standard
-command to restart it. Detecting whether file is outdated or not is based on a
-simple idea. If application has loaded in memory any version of a file
-which is provided by any package updated since system was booted up, tracer
-consider this application as outdated.
+%description %{_description}
 
-%description -n python3-%{name}
-Tracer determines which applications use outdated files and prints them. For
-special kind of applications such as services or daemons, it suggests a standard
-command to restart it. Detecting whether file is outdated or not is based on a
-simple idea. If application has loaded in memory any version of a file
-which is provided by any package updated since system was booted up, tracer
-consider this application as outdated.
+%description -n python2-%{name} %{_description}
+
+Python 2 version.
+
+%description -n python3-%{name} %{_description}
 
+Python 3 version.
 
 %prep
-%setup -q
+%autosetup -n %{name}-%{name}-%{version}-%{rel}
```

Comment:
I will not argue over `%setup` vs `%autosetup`, cause I personally don't care. However why this particular argument for `-n`? It has no problem while building from tito, but rpmbuild fails on that.

I would suggest

    %autosetup -n %{name}-%{version}